tox.ini

```diff
@@ -0,0 +1,18 @@
+[tox]
+minversion = 3.15
+envlist = py310,py39,py38
+
+[testenv]
+no_package = true
```

Comment:
This is because we don't distribute any Python package in this repo. It's solely docs.